src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java

```diff
@@ -225,49 +230,184 @@ public String toString() {
+         * @since {@link RemoteApiVersion#UNKNOWN_VERSION}
+         */
+        @CheckForNull
+        @JsonProperty("Dead")
+        private Boolean dead;
+
+        /**
+         * @since < {@link RemoteApiVersion#VERSION_1_16}
+         */
+        @CheckForNull
         @JsonProperty("Pid")
         private Integer pid;
 
+        /**
+         * @since < {@link RemoteApiVersion#VERSION_1_16}
+         */
+        @CheckForNull
         @JsonProperty("ExitCode")
         private Integer exitCode;
 
+        /**
+         * @since {@link RemoteApiVersion#VERSION_1_17}
+         */
+        @CheckForNull
+        @JsonProperty("Error")
+        private String error;
+
+        /**
+         * @since < {@link RemoteApiVersion#VERSION_1_16}
+         */
+        @CheckForNull
         @JsonProperty("StartedAt")
         private String startedAt;
 
+        /**
+         * @since {@link RemoteApiVersion#VERSION_1_17}
+         */
+        @CheckForNull
         @JsonProperty("FinishedAt")
         private String finishedAt;
 
-        public Boolean isRunning() {
+
+        /**
+         * See {@link #status}
+         */
+        @CheckForNull
+        public String getStatus() {
+            return status;
+        }
+
+        /**
+         * See {@link #running}
+         */
+        @CheckForNull
+        public Boolean getRunning() {
             return running;
         }
 
-        public Boolean isPaused() {
+        /**
+         * See {@link #paused}
+         */
+        @CheckForNull
+        public Boolean getPaused() {
             return paused;
         }
 
+        /**
+         * See {@link #restarting}
+         */
+        @CheckForNull
+        public Boolean getRestarting() {
+            return restarting;
+        }
+
+        /**
+         * See {@link #oomKilled}
+         */
+        @CheckForNull
+        public Boolean getOOMKilled() {
+            return oomKilled;
+        }
+
+        /**
+         * See {@link #dead}
+         */
+        @CheckForNull
+        public Boolean getDead() {
+            return dead;
+        }
+
+        /**
+         * See {@link #pid}
+         */
+        @CheckForNull
         public Integer getPid() {
             return pid;
         }
 
+        /**
+         * See {@link #exitCode}
+         */
+        @CheckForNull
         public Integer getExitCode() {
             return exitCode;
         }
 
+        /**
+         * See {@link #error}
+         */
+        @CheckForNull
+        public String getError() {
+            return error;
+        }
+
+        /**
+         * See {@link #startedAt}
+         */
+        @CheckForNull
         public String getStartedAt() {
             return startedAt;
         }
 
+        /**
+         * See {@link #finishedAt}
+         */
+        @CheckForNull
         public String getFinishedAt() {
             return finishedAt;
         }
 
         @Override
+        public boolean equals(Object o) {
```

Comment:
@marcuslinke please check, btw why nested classes are used? Can they be extracted into model objects?